homeassistant/components/wiser/translations/de.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Please don't include other languages. Remove this file.
Translations can be managed using Lokalise after this PR is merged.

Reply:
This has been done by @msp1974